debian/changelog

```diff
@@ -1,3 +1,14 @@
+libfm-qt (1.1.0-0ubuntu7~ppa2) jammy; urgency=medium
+
+  * Backport to Jammy.
+  * Updated Lubuntu Developers copyright.
+  * Updated symbols file
+  * Removed Repository-Browser field from debian/upstream/metadata.
```

Comment:
Why did you remove it from there?

Reply:
Lintian griped about it, and in previous packages the exact identical data existed in the field above, so I've just been systematically removing that field for Lintian's sake. I get the feeling that *might* be bad? If so, we may have to redo a lot of my earlier backports.

Reply:
Added details.